duckduckgo/macos-browser
# Add content scope code sharing

Task/Issue URL: https://app.asana.com/0/0/1201621593347477/f
Tech Design URL:
CC:

**Description**:

- Loads in shared code from the shared code repo: https://github.com/duckduckgo/content-scope-scripts/
- Only loads the GPC feature for now but enabling fingerprinting is just a config change.
- Requires: duckduckgo/BrowserServicesKit#45

**Steps to test this PR**:
1. Load up the page and check for navigator.globalPrivacyControl in the DOM
2. Change the toggle and check for 'false' value.

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
